app/views/posts/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Details of the post<%= @post_id %> by user<%= @user_id %></h1>
\ No newline at end of file
+<%= stylesheet_link_tag 'posts_styles'%>
+
+<h1>Show more Details of Post by <%= @post.author.name %></h1>
+<% if @post %>
+    <section class='post-list'>
+       <article class='posts box-border'>
+        <h2 class='post-title'>"<%= @post.title %>" by <%= @post.author.name %></h2>
+          <br>
+          <p><%= @post.text %></p>
+          <p class='post-reactions'><%= "Comments: #{@post.comments_counter}, Likes: #{@post.likes_counter}"%></p>
+        </article>
+         <div class='comments box-border'>
+        <h3>Comments</h3>
+        <% @post.recent_comments.each do |comment| %>
+          <div class='comment'>
+            <p class='user-comment'><%= comment.author.name %>: </p>
+            <p class='comment-text'><%= comment.text %></p>
+          </div>
```

Comment:
- [ ] Dear @Iqbal-Elham, on the Post show page we need displays post with all comments listed instead of just the `recent_comments`. Please list all the comments for the post here instead of the recent comments.

Reply:
Thank you for this valuable feedback! @TadesseDev ❤️